bangc-ops/mlu_op.h

```diff
@@ -3776,13 +3776,266 @@ mluOpStatus_t MLUOP_WIN_API mluOpFill(
     mluOpHandle_t handle, const mluOpPointerMode_t pointer_mode,
     const void *value, const mluOpTensorDescriptor_t output_desc, void *output);
 
+
+// Group:RoiawarePool3d
+/*!
+ * @brief Returns in \b workspace_size the size of the MLU memory that is used as an extra
+ * workspace to optimize the RoiawarePool3dForward operation.
+ *
+ * The size of extra workspace is based on the given information of the RoiawarePool3dForward
+ * operation, including the input tensor descriptors \b pts_desc.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices and queues in the
+ * RoiawarePool3dForward operation. For detailed information, see ::mluOpHandle_t.
+ * @param[in] rois_desc
+ * Descriptor of rois, containing the dimension and layout of the rois tensor.
```

Comment:
The descriptor of ROI, which contains the dimension and layout of the ROI tensor.